install_config/cluster_metrics.adoc

```diff
@@ -12,307 +12,433 @@ toc::[]
+The deployer template parameter options and their defaults are listed above in the *_metrics.yaml_* file. If required, you can override these values when creating the *metrics deployer*.
 
-Add the *cluster-reader* role to the *Heapster* service account:
+For example, access to the master server is configured by the `*MASTER_URL*`
+parameter. The default value is `https://kubernetes.default.svc:443`, however
+this likely needs to be overridden and configured for your environment. Failing
+to set this parameter properly may result in timeout errors in the deployer
+logs. You may also want to override the value for `*IMAGE_PREFIX*` to suit your
+deployment.
 
-----
-$ oadm policy add-cluster-role-to-user \
-    cluster-reader \
-    system:serviceaccount:default:heapster
-----
+However, the only required parameter is `*HAWKULAR_METRICS_HOSTNAME*`. This
+value is required when creating the deployer because it specifies the hostname
+for the Hawkular Metrics
+link:../architecture/core_concepts/routes.html[route]. You will need to know the
+value of `*HAWKULAR_METRICS_HOSTNAME*` when
+link:#accessing-metrics-in-the-console[configuring the console] for metrics
+access.
+
+== Deploying the Metric Components
 
-The following is the definition of the *Heapster* replication controller:
+Since deploying and configuring all the metric components is handled by the
+*metrics deployer*, you can simply deploy everything in one step.
 
+The following examples show you how to deploy metrics with and without
+persistent storage using the default template parameters. Optionally, you can
+specify any of the
+link:../install_config/cluster_metrics.html#deployer-template-parameters[template parameters]
+when calling these commands.
+
+.Deploying with Persistent Storage
 ====
-[source,yaml]
+The following command sets the Hawkular Metrics route to use
+`hawkular-metrics.example.com` and is deployed using persistent storage.
+
+You must have a persistent volume of sufficient size available.
+
+[options="nowrap"]
 ----
-apiVersion: "v1"
-kind: "List"
-items:
-  -
-    apiVersion: "v1"
-    kind: "ReplicationController"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "heapster"
-      name: "heapster"
-    spec:
-      replicas: 1
-      selector:
-        provider: "fabric8"
-        component: "heapster"
-      template:
-        metadata:
-          labels:
-            provider: "fabric8"
-            component: "heapster"
-        spec:
-          containers:
-            -
-              args:
-                - "-source=kubernetes:https://kubernetes.default.svc?auth=&insecure=true&useServiceAccount=true"
-                - "-sink=influxdb:http://influxdb-monitoring:8086"
-              image: "kubernetes/heapster:v0.17.0"
-              name: "heapster"
-          serviceAccount: "heapster"
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel \
```

Comment:
I don't know where that came from, its wasn't in the original PR I submitted: https://github.com/mwringe/openshift-docs/blob/753365651e1533926a059d8583ad55f7f4dcc438/admin_guide/cluster_metrics.adoc

You don't need IMAGE_PREFIX or IMAGE_VERSION here, it should be configured to just use the defaults (which is `latest` and not `devel`: https://hub.docker.com/r/openshift/origin-metrics-deployer/tags/